tests/util/build_network_protocol_files.py

```diff
@@ -0,0 +1,169 @@
+    result += encode_data(respond_header_blocks)
+    result += encode_data(coin_state)
+    result += encode_data(register_for_ph_updates)
+    result += encode_data(respond_to_ph_updates)
+    result += encode_data(register_for_coin_updates)
+    result += encode_data(respond_to_coin_updates)
+    result += encode_data(coin_state_update)
+    result += encode_data(request_children)
+    result += encode_data(respond_children)
+    result += encode_data(request_ses_info)
+    result += encode_data(respond_ses_info)
+    return result
+
+
+def get_harvester_protocol_bytes() -> bytes:
+    result = b""
+    result += encode_data(pool_difficulty)
+    result += encode_data(harvester_handhsake)
+    result += encode_data(new_signage_point_harvester)
+    result += encode_data(new_proof_of_space)
+    result += encode_data(request_signatures)
+    result += encode_data(respond_signatures)
+    result += encode_data(plot)
+    result += encode_data(request_plots)
+    result += encode_data(respond_plots)
+    return result
+
+
+def get_introducer_protocol_bytes() -> bytes:
+    result = b""
+    result += encode_data(request_peers_introducer)
+    result += encode_data(respond_peers_introducer)
+    return result
+
+
+def get_pool_protocol_bytes() -> bytes:
+    result = b""
+    result += encode_data(authentication_payload)
+    result += encode_data(get_pool_info_response)
+    result += encode_data(post_partial_payload)
+    result += encode_data(post_partial_request)
+    result += encode_data(post_partial_response)
+    result += encode_data(get_farmer_response)
+    result += encode_data(post_farmer_payload)
+    result += encode_data(post_farmer_request)
+    result += encode_data(post_farmer_response)
+    result += encode_data(put_farmer_payload)
+    result += encode_data(put_farmer_request)
+    result += encode_data(put_farmer_response)
+    result += encode_data(error_response)
+    return result
+
+
+def get_timelord_protocol_bytes() -> bytes:
+    result = b""
+    result += encode_data(new_peak_timelord)
+    result += encode_data(new_unfinished_block_timelord)
+    result += encode_data(new_infusion_point_vdf)
+    result += encode_data(new_signage_point_vdf)
+    result += encode_data(new_end_of_sub_slot_bundle)
+    result += encode_data(request_compact_proof_of_time)
+    result += encode_data(respond_compact_proof_of_time)
+    return result
+
+
+def get_protocol_bytes() -> bytes:
+    return (
+        get_farmer_protocol_bytes()
+        + get_full_node_bytes()
+        + get_wallet_protocol_bytes()
+        + get_harvester_protocol_bytes()
+        + get_introducer_protocol_bytes()
+        + get_pool_protocol_bytes()
+        + get_timelord_protocol_bytes()
+    )
+
+
+if __name__ == "__main__":
+    filename = get_network_protocol_filename()
+    data = get_protocol_bytes()
+    if os.path.exists(filename):
+        print("Deleting old file.")
+        os.remove(filename)
+    f = open(filename, "wb")
+    f.write(data)
```

Comment:
it's not clear to me why you're saving these messages to a file. It just seems to complicate the tests. Why not just call these functions directly in the tests? one at a time.